packages/integration-karma/scripts/karma-configs/base.js

```diff
@@ -88,6 +90,9 @@ module.exports = (config) => {
         // The grep flag is consumed at runtime by jasmine to filter what suite to run.
         client: {
             args: [...config.client.args, '--grep', GREP],
+            jasmine: {
+                random: false, // required for clean-dom/index.spec.js to run in consistent order
```

Comment:
Randomized test ordering prevents implicit coupling between tests. We have been bitten in the past with Jest (test order is fixed in Jest) where the test started breaking when focussing on one of them.

I would strongly advocate keeping this in place.

Reply:
OK, I'll try @ekashida's suggestion as an alternative to setting `random: false`. I agree that tests should be idempotent; that's probably why Jasmine made this the default.

Reply:
Done